packages/design-system/src/components/Label/Label.stories.tsx

```diff
@@ -23,15 +32,4 @@ export default meta;
 
 type Story = StoryObj<typeof Label>;
 
-export const LabelDefault: Story = {};
-
-export const InverseLabel: Story = {
-  args: {
-    inversed: true,
-  },
-  parameters: {
-    // Must supply `layout: 'fullscreen'` when we use `onDark: true`
-    onDark: true,
-    layout: 'fullscreen',
-  },
-};
+export const Default: Story = {};
```

Comment:
updating the `fieldId` when set to `legend` i noticed the `for` attr became an `htmlfor` attr. i can't find [`htmlfor` in MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend) but maybe i'm misunderstanding how legend works

Reply:
`htmlFor` is a Reactism because `for` is a JavaScript keyword. I'll have to investigate why that's happening.

Reply:
It's supposed to be translated as `for` when rendered to the DOM

Reply:
Seems like it doesn't do it in the React version. I think Preact actually recognizes that `for` is not a valid attribute for `<legend>`. Either we should update our component logic to not add the `for` attribute when it's a legend, or we just rely on people knowing that they're not supposed to use the combination.

Reply:
[Updated the component](https://github.com/CMSgov/design-system/pull/2841/commits/3d3b14f71b9e823247cbfbedb5717861adf42e0a)